src/components/ProfileUserAvatar/ProfileUserAvatar.js

```diff
@@ -0,0 +1,45 @@
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
+import Image from 'next/image'
+import { useState, useCallback } from 'react'
+
+import useSelectorWithProps from '~/hooks/useSelectorWithProps'
+import { selectAvatarByUserId, withCurrentUserId } from '~/store/selectors'
+
+import UploadAvatarModal from '../UploadAvatarModal'
+import styles from './ProfileUserAvatar.module.scss'
+
+function ProfileUserAvatar () {
+  const userAvatar = useSelectorWithProps({ size: 170 }, withCurrentUserId(selectAvatarByUserId))
+
+  const [showUploadAvatar, setShowUploadAvatar] = useState(false)
+
+  const handleToggleUploadAvatar = useCallback(() => {
+    setShowUploadAvatar(!showUploadAvatar)
+  }, [showUploadAvatar])
+
+  return (
+    <>
+      <div className={styles.userAvatar}>
+        <div className="avatar xl">
+          <Image
+            unoptimized
+            alt="User's avatar"
+            height={170}
+            src={userAvatar}
+            width={170} />
+        </div>
+        <div className={styles.userAvatarEdit}>
```

Comment:
I'm not exactly sure why that graphical bug exists here, but I will pull down the PR locally and load it up to see. Will update this with a change suggestion when I get it working!